.github/workflows/python.yml

```diff
@@ -28,6 +28,10 @@ jobs:
     - name: sbt install
       run: |
         sbt publishLocal
+    - name: Start docker-compose
```

Comment:
using mlflow via os multiple process is sufficient, I think

Reply:
That sounds like a good idea, I may test it with another PR.